test/browsers/devicefarm-mobile.test.js

```diff
@@ -0,0 +1,25 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+const MobileBrowserCreator = require('../../src/browsers/devicefarm-mobile').default;
+
+const seleniumUrl = 'http://localhost:4444/wd';
+
+describe('Mobile Devicefarm browserCreator', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
```

Comment:
I don't see any mocks being used. Is this needed?

Reply:
Oops, copy-paste mistake!